### PR TITLE
Create ui-c8y-1020-32-2-fix-running-karma-tests-against-ngx-components-for-layered-products-7320.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-61312](https://cumulocity.atlassian.net/browse/MTM-61312)
Original PR: [7320](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7320)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-32-2-fix-running-karma-tests-against-ngx-components-for-layered-products-7320.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: fix running karma tests against ngx-components for layered products (#7320)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-32-2-fix-running-karma-tests-against-ngx-components-for-layered-products-7320.md]: The Claude AI suggested to use the following title:
```suggestion
title: Fix running Karma tests against ngx-components for layered products
```

File: content/change-logs/application-enablement/ui-c8y-1020-32-2-fix-running-karma-tests-against-ngx-components-for-layered-products-7320.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61312
+version: 1020.32.2
+---
+fix running karma tests against ngx-components for layered products (#7320)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-32-2-fix-running-karma-tests-against-ngx-components-for-layered-products-7320.md]: The Claude AI suggested to use the following body:
```suggestion
Previously, running Karma tests against ngx-components for layered products was not working correctly. With this change, the issue has been resolved by fixing the configuration for running Karma tests against ngx-components. Layered products can now successfully run their Karma tests against ngx-components without encountering any problems.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1020-32-2-fix-running-karma-tests-against-ngx-components-for-layered-products-7320.md (hunk 1 above)
Comment (reey) [reply content/change-logs/application-enablement/ui-c8y-1020-32-2-fix-running-karma-tests-against-ngx-components-for-layered-products-7320.md]:
```suggestion
title: Fix running Karma tests against @c8y/ngx-components library
```

File: content/change-logs/application-enablement/ui-c8y-1020-32-2-fix-running-karma-tests-against-ngx-components-for-layered-products-7320.md (hunk 2 above)
Comment (reey) [reply content/change-logs/application-enablement/ui-c8y-1020-32-2-fix-running-karma-tests-against-ngx-components-for-layered-products-7320.md]:
```suggestion
Version 1020.26.0 of the @c8y/ngx-components library introduced an issue which made it impossible for consumers of this library to perform Karma tests against it. This issue has been addressed now.
```

**Review by reey — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-32-2-fix-running-karma-tests-against-ngx-components-for-layered-products-7320.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fix running Karma tests against @c8y/ngx-components library
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-32-2-fix-running-karma-tests-against-ngx-components-for-layered-products-7320.md]:
```suggestion
title: Fixed issue with Karma tests against @c8y/ngx-components library
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**